_posts/2018-12-12-cockpit-184.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Cockpit 184
+author: mvollmer
+date: 2018-12-12
+tags: cockpit linux
+slug: cockpit-184
+category: release
+summary: Cockpit with machines and storage improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 184.
+
+### Machines: Dialog and tab layout is now responsive
```

Comment:
How about something like "These UI elements now adjust their layout to small and mobile browser windows", and using a screenshot like

![machines-create-vm-mobile](https://user-images.githubusercontent.com/200109/49949968-2b33ea80-fef7-11e8-8751-dcb787bdfa3c.png)